features/management_students/deleteStudent.feature

```diff
@@ -59,4 +59,25 @@ Feature: Set of tests to delete students
 		Then forgot password email was not sent to '<studentUsername>'
 		Examples:
 			| userRole | adminUsername                | adminPassword  | studentUsername                  |  deletedUserRole |
-			| admin    | kai.admin.qa@schul-cloud.org | Schulcloud1qa! | amelia.strobl.qa@schul-cloud.org |  student         |
\ No newline at end of file
+			| admin    | kai.admin.qa@schul-cloud.org | Schulcloud1qa! | amelia.strobl.qa@schul-cloud.org |  student         |
+
+	@classWithTwoStudentsAndOneGetsDeleted @deletionConcept
+	Scenario Outline: As a user, I want to delete a student from a class and he will be no longer visible in class members
+		Given <userRole> logs in with email '<adminUsername>' and password '<password>'
+		And <userRole> goes to management
+		And <userRole> goes to class management
+		When <userRole> creates class with custom name '<customClassName>'
```

Comment:
I think now after merge of my PR with sort of class years, You should use that user is creating a class with given year like 2020/21